runtime-builder/tests/integration/go-get.sh

```diff
@@ -13,7 +13,7 @@ echo "Fetching dependencies..."
 cd src/app
 vendor_dir="$(pwd -P)/vendor"
 mkdir "${vendor_dir}"
-go get -d -tags appenginevm
+set +e; go get -d -tags appenginevm
```

Comment:
Why do we need it here?

Reply:
It errors out when trying to get app/internal/foo and stops the script from continuing.

Reply:
Got it, thanks.